ocfweb/docs/docs/about.md

```diff
@@ -67,3 +67,4 @@ this website.
 [decal]: https://decal.ocf.berkeley.edu
 [csua]: http://csua.berkeley.edu
 [minutes]: https://www.ocf.berkeley.edu/~staff/bod/
+[tech]: https://www.ocf.berkeley.edu/docs/staff/techtalks/
```

Comment:
This URL shouldn't be hardcoded. We have a wiki-like syntax for internal links; see the other docs pages for examples.